org.mwc.debrief.core/src/org/mwc/debrief/core/ContextOperations/GeneratePasteRepClipboard.java

```diff
@@ -95,12 +95,10 @@ public IStatus undo(final IProgressMonitor monitor, final IAdaptable info)
     }
   }
 
-  private static Action doPasteAction;

```

Comment:
@ayeshare - we didn't need to keep a copy of the action, all we need to do is create the action, and add it to the parent.